### PR TITLE
Remove comment from CLI command codeblock

### PR DESCRIPTION
**Pull request by natalyjazzviolin**
Issue:
When copying the CLI command for installing Storybook on React, the comment is copied also. This defeats the purpose of the handy copy button, as the user then needs to manually delete the comment in their terminal.

## What I did
Removed the comment from the CLI command code block.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by shilman — COMMENTED** [review]
Nice improvement, thanks! I see there are a bunch of other places in the code that use the same pattern. Any chance you can clean those up as well? cc @jonniebigodes

**Comment by jonniebigodes** [comment]
@natalyjazzviolin thanks for the time and effort you've put into this pull request and your willingness to help improve the documentation. Let me know once you've addressed the feedback that @shilman mentioned. Or if you have any questions feel free to ask.

Hope you have a great day!

Stay safe

**Comment by natalyjazzviolin** [comment]
Working on it, I'm glad this is helpful! Question: should codeblocks like these be split up in favor of using the copy button, or should I just leave them alone?
This one is from here: https://storybook.js.org/docs/react/configure/webpack

Screen Shot 2022-02-28 at 2 31 10 PM

**Comment by jonniebigodes** [comment]
@natalyjazzviolin and we're glad that you're helping us with it. To answer your question, as a general rule of thumb JSON, shell, bash scripts are left as is in the docs, to avoid maintenance costs and keep it at a functional level.

**Comment by natalyjazzviolin** [comment]
@jonniebigodes oh, let me rephrase the question:

I'm working on making sure a shell script has no comments inside of it, so the reader can click the 'Copy' button and paste it into their terminal. I see that some shell scripts have multiple commands. Should I split these up to make them copyable -
For development mode run this:
`yarn start-storybook --debug-webpack`
For production mode run this:
`yarn build-storybook --debug-webpack`

Or should I focus solely on oneliners and not worry about multiline blocks?

**Comment by jonniebigodes** [comment]
@natalyjazzviolin thanks for taking the time to follow up on this and continue working on updating the documentation. I've spotted some items that I'll follow up with you at a later date once you wrap up the updates. Let me know once you've finished and I'll review it with you, sound good?

Hope you have a fantastic weekend!

Stay safe

**Comment by jonniebigodes** [comment]
@natalyjazzviolin one thing that I would like to bring up to you about this pull request. It seems that you're pushing your next branch against our next branch and this might cause some issues as probably your fork might not be up to date or something of that nature. If you wouldn't mind and sorry for not bringing this to your attention earlier, if you could close this pull request and make a new one in a separate branch on your end I'd appreciate it. As to avoid the possible issues I've mentioned above and we'll take it from there! As the work you've done with this pull request just needs a couple of finishing touches and we can merge it once said pr in said branch is up.

Let me know and we'll go from there.

Hope you have a great weekend! Looking forward to the updated pull request 🙂.

Stay safe.
